.gitignore

```diff
@@ -0,0 +1 @@
+venv
```

Comment:
Great practice to create a gitignore. Beautiful!
Make sure you maintain it as your projects grow :)